src/print.rs

```diff
@@ -132,12 +132,45 @@ pub fn explain(args: ArgMatches) {
 }
 
 fn compute_modules<'a>(context: &'a Context) -> Vec<Module<'a>> {
-    let mut prompt_order: Vec<&str> = Vec::new();
+    enum Mod<'a> {
+        Builtin(&'a str),
+        Custom(&'a str),
+    }
+
+    struct DebugCustomModules<'tmp>(&'tmp toml::value::Table);
+
+    impl Debug for DebugCustomModules<'_> {
+        fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+            f.debug_list().entries(self.0.keys()).finish()
+        }
+    }
+
+    let mut prompt_order: Vec<Mod> = Vec::new();
 
     // Write out a custom prompt order
     for module in context.config.get_root_config().prompt_order {
         if ALL_MODULES.contains(&module) {
-            prompt_order.push(module);
+            // Write out a module if it isn't disabled
+            if !context.is_module_disabled_in_config(module) {
+                prompt_order.push(Mod::Builtin(module));
+            }
+        } else if module.starts_with("custom.") {
+            // Write out a custom module if it isn't disabled (and it exists...)
+            match context.is_custom_module_disabled_in_config(&module[7..]) {
+                Some(true) => (), // Module is disabled, we don't add it to the prompt
+                Some(false) => prompt_order.push(Mod::Custom(&module[7..])),
+                None => match context.config.get_custom_modules() {
+                    Some(modules) => log::debug!(
+                        "Expected prompt_order to contain custom value from {:?}. Instead received {}",
+                        DebugCustomModules(modules),
+                        module,
+                    ),
+                    None => log::debug!(
+                        "Expected prompt_order to contain a custom value, but none was provided. Instead received {}",
```

Comment:
I wasn't totally clear on what the log message was saying here. Perhaps something along these lines should be used:
`prompt_order contains custom module "{}", but no configuration was provided. Configuration for the following modules were provided: {}`

Reply:
Once again, good description! Thank you.